_includes/scenarios/integrate/establish-a-relationship-to-another-identity.md

```diff
@@ -0,0 +1,168 @@
+
+### Accept the RelationshipRequest
+
+If the Templator accepts the RelationshipRequest, the `status` of the data object of type [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) will change from `"Pending"` to `"Active"` and therefore an active Relationship between the Templator and the Requestor will be established. In addition, the `transport.relationshipChanged` [Connector event]({% link _docs_integrate/connector-events.md %}) will be triggered. To accept the RelationshipRequest, consult the [Accept RelationshipChange]({% link _docs_use-cases/use-case-transport-accept-relationship-change.md %}) use case description and specify the `id` of the Relationship and the `id` of the associated RelationshipChange.
+
+For rejecting the RelationshipRequest and therefore not establishing an active Relationship between the Templator and the Requestor, take a look at the documentation of the [Reject RelationshipChange]({% link _docs_use-cases/use-case-transport-reject-relationship-change.md %}) use case.
+{: .notice--info}
+
+### Be informed about the acceptance of the RelationshipRequest
+
+Assuming the Requestor is a Connector, it must [synchronize the updates of the Backbone]({% link _docs_use-cases/use-case-transport-synchronize-updates-of-backbone.md %}) after the Templator has accepted the RelationshipRequest. The synchronization causes the `transport.relationshipChanged` [Connector event]({% link _docs_integrate/connector-events.md %}) to be triggered and the `result` of the response after synchronization shows in particular that the `status` of the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) has been changed from `"Pending"` to `"Active"` and that the `status` of the associated [RelationshipChange]({% link _docs_integrate/data-model-overview.md %}#relationshipchange) with `"Creation"` as `type` has been changed from `"Pending"` to `"Accepted"`. Now the Requestor is informed that the Templator has accepted the RelationshipRequest and therefore an active Relationship has been established between them. If the Requestor is an App user instead, they are informed about the acceptance of the RelationshipRequest analogously, without having to synchronize manually.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/650bd1ea-e0fd-489e-ab26-e7fe201ef59e" id="~uUgN3n5x3eT"></iframe></div>
+
+## What's next?
+
+After an active Relationship between the two Identities is established, they are able to share information with each other. For example, they can exchange Messages. A possible scenario that demonstrates the usefulness of the ability of a Connector to send a Message to another Identity with which it has an active Relationship is described in the [Requests over Messages]({% link _docs_integrate/requests-over-messages.md %}) scenario documentation.
```

Comment:
" demonstrates the usefulness of the ability of a Connector to send a Message" -> "demonstrates how a Connector can send a Message"
"usefulness of the ability" sounds rather cumbersome. I don't think it requires a motivation as to why we provide the feature of sending Messages.